index.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'yaml'
+require 'faker'
+require 'date'
+
+require_relative 'entities/author'
+require_relative 'entities/book'
+require_relative 'entities/reader'
+require_relative 'entities/order'
+
+require_relative 'generator_fake_data'
+require_relative 'create_database'
+require_relative 'statistic'
+require_relative 'validation/errors'
+
+# main class
+class Library
```

Comment:
why library class in index.rb ?
move to separate file

Reply:
corrected